app/api/models.py

```diff
@@ -0,0 +1,82 @@
+from enum import Enum
+
+from sqlalchemy import ForeignKey, Integer, String
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from app.core.constance import CATEGORY_NAME_LENGTH, STRING_LENGTH
+from app.core.db import Base
+
+
+class SpeakingSkillEnum(str, Enum):
+    begginer = 'begginer'
+    pre_intermidiate = 'pre-intermidiate'
+    intermidiate = 'intermidiate'
+    pre_advanced = 'pre-advanced'
+    advanced = 'advanced'
+    native = 'native'
+
+
+class TagsEnum(str, Enum):
+    pronunciation = 'pronunciation'
+    speaking = 'speaking'
+    comprehension = 'comprehension'
+    listenig = 'listenig'
+    writing = 'writing'
+    reading = 'reading'
+
+
+class DurationEnum(str, Enum):
```

Comment:
Haven't work much with enums but since duration is in intergers perhaps `int` would be more appropriate?